src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 3.4.0
 -----
 
+ * added a setToken() method to the SwitchUserEvent class to allow to replace the created token while switching users
```

Comment:
the method and class names should be enclosed with backticks:

```markdown
 * added a `setToken()` method to the `SwitchUserEvent` class to allow to replace the created token while switching users
```

Reply:
Fixed